configs/gateway/prepare-certbot.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# DOMAIN_NAME=$1 # "api2.gifts.hotdeals.dev"
+# EMAIL=$2 # "dealer@hotdeals.dev"
+
+
+if [ -z ${1+x} ]; then
+    echo "[ERROR]: The parameter in slot '1' for 'DOMAIN_NAME' is unset!";
+    exit 1;
+    else DOMAIN_NAME=$1;
+fi;
+echo "[INFO]: Domain name is: $DOMAIN_NAME"
+
+if [ -z ${2+x} ]; then
+    echo "[ERROR]: The parameter in slot '2' for 'EMAIL' is unset!";
+    exit 1;
+    else EMAIL=$2;
+fi;
+echo "[INFO]: Email is: $EMAIL"
+
+# Generate the tokens
+echo "Generating tokens..."
+certbot --nginx -d $DOMAIN_NAME --non-interactive --agree-tos -m $EMAIL
+
+
+# Add cronjob for renewing certs - 30 days
+echo "[INFO]: Creating cron job..."
+echo "0 12 * * * /usr/bin/certbot renew --quiet" > mycron
+crontab mycron
+
+echo "[INFO]: Certbot script completed."
```

Comment:
Script that can be executed on a deployed nginx container, its usage is <script_name.sh> "hostname" "real_email"

the "hostname" variable MUST match the name of the config inside nginx/conf.d